config/api.yml

```diff
@@ -848,6 +848,8 @@
       - :name: read
         :identifier: cloud_volume_snapshot_view
       :post:
+      - :name: revert
+        :identifier: cloud_volume_snapshot_revert # doesn't exist yet
```

Comment:
@h-kataria are you able to advise on this? Unfortunately I don't have any development instances that are snapshot-able - is it possible to revert an instance snapshot? Or is that tied into the backup/restore flow?